pkg/plugin/validator_test.go

```diff
@@ -169,6 +177,17 @@ func TestMatchIssue(t *testing.T) {
 			if diff := testutil.DiffErrString(gotErr, tc.wantErrSubstr); diff != "" {
 				t.Errorf("Process(%+v) got unexpected error substring: %v", tc.name, diff)
 			}
+			if gotErr != nil {
```

Comment:
this should be `tc.wantErrSubstr != ""`.

Reply:
Done